### DISCUSSION TITLE
"Failed to process the payload. Ensure that it is less than 2 MB and contains valid UTF-8." occur error

### DISCUSSION THREAD

**Discussion by Kang0228** [deepgram/community#531] [General help] [answered]
I am using local file transcription.
However, I'm getting an error like the one above, and I'm wondering why it only supports 2MB. Is there a solution?
request_id : "95b8e767-250e-4fae-ab65-04b24de8ecdb"

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by nikolawhallon** [reply, marked as answer]
Can you share how the request is being made? It looks like this is a request with a json body which is meant to supply Deepgram with a URL to fetch the audio from, rather than a request with a binary body containing the audio itself (the latter of which has a size limit in the GBs!).

**Reply by Kang0228** [reply]
That's a great solution, thanks!

**Comment by davidvonthenen** [comment]
I think I remember something where if you specify the mine type, it will give that error. If you are using specifying one, please try and remove it.

**Comment by davidvonthenen** [comment]
@Avalbard were you able to get this to work?

**Reply by Kang0228** [reply]
I made a stupid mistake.
I coded the localfile method to work as a URL.
That's why I had the 2MB issue.
I swapped the two and it worked fine.
